tests/e2e/helm.go

```diff
@@ -164,7 +164,7 @@ func (helmDescriptor *helmDescriptor) installHelmChart(kubectlOptions k8s.Kubect
 		fixedArguments := []string{
 			"--create-namespace",
 			"--atomic",
-			"--debug",
+			//"--debug",
```

Comment:
Question: Is this comment intended? Or do we actually want to remove this debug flag?

Reply:
This generates lot of logs. Same belongs for cluster resource snapshot.
I think this should be investigated that are these usable for anything because these causes the output  truncate on github

Reply:
I put back the debug because the investigatnion is not part of this PR.